app/main/class-comment-mention.php

```diff
@@ -334,7 +337,7 @@ function() {
 	 * @return void
 	 */
 	public function cmt_mntn_check_mention( $comment_ID, $comment_status, $comment_data ) {
-		if ( function_exists( 'cmt_mntn_pro_membership_status' ) && ! cmt_mntn_pro_membership_status() ) {
+		if ( function_exists( 'cmt_mntn_pro_membership_status' ) && ! cmt_mntn_pro_membership_status() || ! cmt_mntn_check_enabled_userroles() ) {
```

Comment:
Can we make a function like "cmt_mntn_can_user_mention" and in that we can have these conditions?